my_site/sessions/sess_2umrptvqqglblcv0rb8s5ah519

```diff
@@ -0,0 +1 @@
+is_logged_in|b:1;
```

Comment:
You shouldn't add the sessions files to github. They could eventually contain critical information, depending on your website organization.